### PR TITLE
Javascript

### PR DESCRIPTION
**Pull request by joymeetme**
### Task: <task name here>

I <clear and concise explanation>
I <clear and concise explanation>

### CODE REVIEW

**Comment by joymeetme** [comment]
i added my javascript Task

File: submissions/ayodeleayobami/javascript/product.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    },
+    {
+        name:"pumps",
+        image:"images/shoe3.jpg",
+        price:"$120"
+    },
+];
+function loadProducts() {
+    products.forEach(function(item) {
+          product =
```
Comment (tegaphilip) [inline submissions/ayodeleayobami/javascript/product.js:20]: you need to declare the variable `product` here. e.g. `var product`. you can also declare it outside the loop. also the lines from `<div class ...` to `</div>` should be wrapped as a string before you can assign to the variable `product`.  Make sure to test properly

See

File: submissions/ayodeleayobami/javascript/product.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        <div class ="product-box">
+            <div class="product-content">
+            <div class="product image">
```
Comment (tegaphilip) [inline submissions/ayodeleayobami/javascript/product.js:23]: class should be `product-image`

File: submissions/ayodeleayobami/javascript/product.html [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+		</div>
+		<div class="search">
+		<form action=""> 
+			<input class="search"  type="text" name="text"
+			   placeholder="search for product" id="search">
+		    <input class="search-button" type="submit" name="submit">
+		</form>
+		</div>
+			<div class="product-box">
+			<div class="product-content">
```
Comment (tegaphilip) [inline submissions/ayodeleayobami/javascript/product.html:24]: according to the task, these items should not be there until the button is clicked please check the task again and confirm. i am assuming that when you load the page, you don't see any products until the button is clicked.

File: submissions/ayodeleayobami/javascript/product.css [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+}
+
+
+/*------------------product-sale CSS-----------*/
+.product-image
+{
+	margin:50px  0;
+}
+.product-image img
+{
```
Comment (beccadaniel) [inline submissions/ayodeleayobami/javascript/product.css:44]: Hello @joymeetme Can you refactor this file as a whole and ensure it is well indented? The indentations are a bit off.

**Review by beccadaniel — CHANGES_REQUESTED** [review]

**Pull request merged**